### PR TITLE
icons: Updated bot icon with single antenna version.

### PR DESCRIPTION
**Pull request by fombalang**
This PR updates the previous bot icon, replacing it with a new single antenna bot icon. 

No tests have been written or updated for this PR as the changes made were minimal and did not involve changing any dart code, just updating the SVG asset file and rebuilding the icon font.

Fixes: #978 

Thank you.

### CODE REVIEW

**Comment by chrisbobbe** [comment]
Please write down specifically where you got the new icon from. The commit message is a good place for that, and the first place someone in the future will look when they want to know where it came from.

File: pubspec.lock [hunk 1]
```diff
@@ -1060,10 +1060,10 @@ packages:
     dependency: "direct dev"
```
Comment (PIG208) [inline pubspec.lock:1060]: Looks like the update to this file is unintended.

**Comment by fombalang** [comment]
> Please write down specifically where you got the new icon from. The commit message is a good place for that, and the first place someone in the future will look when they want to know where it came from.

Yes, the icon was gotten from the `zulip` repository from this PR about updating the icon. https://github.com/zulip/zulip/pull/31673
I downloaded the SVG file from that repository and used it.

File: pubspec.lock (hunk 1 above)
Comment (fombalang) [reply pubspec.lock:1060]: Yes, it was unintended. It was changed when I ran a `flutter pub get`

**Comment by fombalang** [comment]
> > Please write down specifically where you got the new icon from. The commit message is a good place for that, and the first place someone in the future will look when they want to know where it came from.
> 
> Yes, the icon was gotten from the `zulip` repository from this PR about updating the icon. [zulip/zulip#31673](https://github.com/zulip/zulip/pull/31673) I downloaded the SVG file from that repository and used it. Specifically from https://github.com/zulip/zulip/blob/main/web/shared/icons/bot.svg

Should I send an updated commit with a description of where I got the icon from or adding a comment here would be fine? @chrisbobbe

**Comment by chrisbobbe** [comment]
Yes, please update the commit. As I said above:

> […] The commit message is a good place for that, and the first place someone in the future will look when they want to know where it came from.

**Comment by fombalang** [comment]
> Yes, please update the commit. As I said above:
> 
> > […] The commit message is a good place for that, and the first place someone in the future will look when they want to know where it came from.

Okay, understood, thank you.

**Comment by fombalang** [comment]
I have updated the commit @chrisbobbe

File: assets/icons/bot.svg
[the remark this answers is not in the document]
Comment (fombalang) [reply assets/icons/bot.svg:1]: Okay, noted, thank you

**Comment by chrisbobbe** [comment]
Ah, I see you've also wrapped the URL onto two lines :) but we do like to keep URLs intact instead of breaking them. So that can be an exception to the wrap-to-68-columns rule.

**Comment by fombalang** [comment]
> Ah, I see you've also wrapped the URL onto two lines :) but we do like to keep URLs intact instead of breaking them. So that can be an exception to the wrap-to-68-columns rule.

Yeah, I was actually confused about what to do with URLs, let me update that then.

**Comment by fombalang** [comment]
> > Ah, I see you've also wrapped the URL onto two lines :) but we do like to keep URLs intact instead of breaking them. So that can be an exception to the wrap-to-68-columns rule.
> 
> Yeah, I was actually confused about what to do with URLs, let me update that then.

Updated

**Comment by chrisbobbe** [comment]
Ah, and one thing I forgot in my previous reviews: since this makes a visible change, please post before/after screenshots. The code and commit message look good to me though, and I'll move this along to Greg for his review (but please do post those screenshots).

**Comment by chrisbobbe** [comment]
Oh, I see you did post screenshots, just on the issue thread instead of here: https://github.com/zulip/zulip-flutter/issues/978#issuecomment-2393209398

In future, please post the screenshots on the PR thread. But I do think those screenshots look good.

**Comment by gnprice** [comment]
Thanks @fombalang for taking care of this, and @chrisbobbe @PIG208 for the previous reviews!

This looks good; merging. I fixed one nit in the commit message:

    -    icons: Update bot icon with single antenna version
    +    icons: Update bot icon with single-antenna version

**Pull request merged**